frontend/models/feedback-task-model/feedback-dataset/FeedbackDataset.model.js

```diff
@@ -0,0 +1,25 @@
+import { Model } from "@vuex-orm/core";
+import { GlobalQuestion } from "../global-question/GlobalQuestion.model";
+import { Record } from "../record/Record.model";
+
+class FeedbackDataset extends Model {
+  static entity = "feedbackDatasets";
+
+  static fields() {
+    return {
+      id: this.uid(),
+      name: this.string(""),
+      guidelines: this.string(""),
+      workspace_id: this.attr(null),
+      workspace_name: this.attr(null),
```

Comment:
Is possible to fetch the workspace through a relationship with the workspace entity?

Reply:
Do you mean updating the orm like this ? 
![image](https://user-images.githubusercontent.com/88380932/232020613-72853963-130a-426c-b464-a8957b565697.png)